Purchases/Purchasing/RCAttributionFetcher.m

```diff
@@ -27,12 +27,36 @@ + (instancetype)sharedManager;
 
 @implementation RCAttributionFetcher : NSObject
 
-- (nullable NSString *)advertisingIdentifier
+- (NSString *)rotateString:(NSString *)string
```

Comment:
we should either specify non null or better yet add `NS_ASSUME_NONNULL_BEGIN`

Reply:
`RCAttributionFetcher.h` already has the `NS_ASSUME_NONNULL_BEGIN`